chaoscenter/upgrade-agents/control-plane/versions/v0.0.0/upgrade.go

```diff
@@ -0,0 +1,135 @@
+package v0_0_0
+
+// remove
+import (
+	"context"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/litmuschaos/litmus/chaoscenter/upgrader-agents/control-plane/pkg/database"
+	log "github.com/sirupsen/logrus"
+)
+
+/* This example upgrade renames auth collection users to client*/
+
+func upgradeUsersCollection(logger *log.Logger, dbClient *mongo.Client, ctx context.Context) error {
+
+	logVersion := log.Fields{
```

Comment:
Can you please follow this kind of logging here: https://github.com/litmuschaos/litmus/pull/4740/files#diff-c3dcc200129e5965fb7dc2655155eb285cfea424297940f911ffe1b6624b8316R62